release-engineering/role-handbooks/branch-manager.md

```diff
@@ -198,36 +201,47 @@ Note:
 
 Builds against the `master` branch are implicitly the next alpha. `gcbmgr` and `anago` automatically finds and increments the current build number.
 
-## Branch Creation
+## Beta Releases
+
+### Branch Creation
+
+During a `beta.0` release e.g., `1.16.0-beta.0`, our release tooling creates a new release branch named `release-x.y`, where `x` and `y` are the major and minor versions of the next release, respectively.
 
-The first time a specific branch is mentioned for the release process instead of the `master` branch is when beta.0 is being cut, for example:
+Behind the scenes `anago` is doing a branch create and push.
 
-`./gcbmgr stage release-1.12 --build-at-head --nomock`
+`prow`’s [`branchprotector`](https://git.k8s.io/test-infra/prow/cmd/branchprotector/README.md) runs once per day and automatically adds [branch protection](https://help.github.com/articles/about-protected-branches/) to any new branch in the `k/k` repo.
```

Comment:
nit: would be better to explicitly spell out `k/k` to `kubernetes/kubernetes` since newcomers also read the handbook :)